tests/testthat/test-get_source_expressions.R

```diff
@@ -414,6 +414,8 @@ param_df$.test_name <- with(param_df, sprintf("%s on expression %d", linter, exp
 patrick::with_parameters_test_that(
   "linters pass with xml_missing() content",
   {
+    skip_if_not_installed("cyclocomp")
```

Comment:
this actually induces 340 skips, we only need one though right?

Reply:
How about

```
if (linter == "backport_linter") {
  ...
} else {
  if (linter == "cyclocomp_linter") {
    skip_if_not_installed("cyclocomp")
  }
  linter <- eval(call(linter))
}
```

Reply:
Relevant again :smile: 

https://github.com/google/patrick/issues/19

Reply:
Good call!